app.js

```diff
@@ -1,148 +1,106 @@
-//Variables para llevar el registro del puntaje y el resultado general
+//Vars to register score and general result
 
 let score = 0;
 let totalQuestions = 5;
-
-// Función para manejar el evento submit del PRIMER formulario
-function handleQuestion1Submit(event) {
+let currentQuestion = 1;
+
+//Functions to handle submit form submission
+function handleQuestionSubmit(
+  event,
+  correctAnswer,
+  correctMessage,
+  incorrectMessage
+) {
   event.preventDefault();
 
   const selectedValue = document.querySelector(
-    'input[name="questionOne"]:checked'
+    `input[name="question${currentQuestion}"]:checked`
   );
 
-  if (selectedValue && selectedValue.value === "b") {
+  if (selectedValue && selectedValue.value === correctAnswer) {
     alert("¡Respuesta correcta!");
```

Comment:
Not part of this review but try to always put string literals inside constants

const CORRECT_ANSWER_COPY = "¡Respuesta correct!"